src/models/abc/world.py

```diff
@@ -45,7 +45,7 @@ def forward(
         next_obs: _tensor_or_any,
         *args: Any,
         **kwds: Any,
-    ) -> tuple[_dist_or_any, _dist_or_any, _tensor_or_any]:
+    ) -> tuple[_dist_or_any, _dist_or_any, _tensor_or_any, _tensor_or_any]:
```

Comment:
ここの変更は何によって生まれたのでしょうか？
```suggestion
    ) -> tuple[_dist_or_any, _dist_or_any, _tensor_or_any]:
```

Reply:
記憶にない変更です...（？？）
もとに戻しておきます